src/apis/user/user.service.ts

```diff
@@ -85,23 +86,32 @@ export class UserService {
     const checkNickName = await this.userRepository.findOne({
       where: { nickname },
     });
-    const checkPhone = await this.userRepository.findOne({
-      where: { phoneNumber },
-    });
+    // const checkPhone = await this.userRepository.findOne({
+    //   where: { phoneNumber },
+    // });
     const checkEmail = await this.userRepository.findOne({
       where: { email },
     });
     if (checkNickName) {
-      throw new NotFoundException('이미 사용 중인 닉네임 입니다.');
+      throw new ConflictException('중복된 닉네임이 존재합니다.');
     } else if (checkEmail)
-      throw new NotFoundException('이미 사용 중인 이메일 입니다.');
+      throw new ConflictException('중복된 이메일 계정이 존재합니다.');
 
-    if (checkPhone) {
-      throw new NotFoundException('이미 사용 중인 휴대폰 번호입니다.');
-    }
+    // if (checkPhone) {
+    //   throw new NotFoundException('이미 사용 중인 휴대폰 번호입니다.');
+    // }
 
+    //이메일 인증이 안되었을 때
     if (isValid !== true || !isValid)
-      throw new BadRequestException('인증이 완료되지 않았습니다.');
+      throw new BadRequestException('인증이 완료되지 않은 이메일입니다.');
+
+    //비밀번호 형식
+    const passwordRule =
+      /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,16}$/;
+
+    if (passwordRule.test(createUserInput.password) !== true) {
```

Comment:
```suggestion
    if (passwordRule.test(createUserInput.password) === false) {
```
개인취향이긴 하지만 부정문을 사용하면 한번 더 생각해야해서 불편한감이 있더라고요

Reply:
> 개인취향이긴 하지만 부정문을 사용하면 한번 더 생각해야해서 불편한감이 있더라고요

이건 비교가 부정적이면 연산자가 긍정이고, 연산자가 긍정이면 비교가 부정이라 ㅋㅋㅋ
test 리턴값이 true, false면 그냥 비교연산자를 제거 하는게 더 깔끔할듯요